pipeline_dp/pld_budget_accounting.py

```diff
@@ -0,0 +1,114 @@
+"""Privacy Loss Distribution (PLD) budget accounting for DP Pipelines."""
+
+from dataclasses import dataclass, field
+from pipeline_dp.aggregate_params import NoiseKind
+from dp_accounting import privacy_loss_distribution
+
+
+@dataclass
+class PLDBudget:
+    _noise_parameter: float = None  # i am not sure how to use this, as noise is calibrated globally?
+
+    @property
+    def noise_parameter(self):
+        if self._noise_parameter is None:
+            raise AssertionError("Noise parameter is not yet defined.")
+        return self._noise_parameter
+
+
+@dataclass
+class RequestedPLDBudget:
+    budget: PLDBudget
+
+
+@dataclass
+class PLDBudgetAccountant:
+    noise: float = 1  # this is where i am storing noise, does this make sense?
+    max_noise: float = 50
+    sensitivity: float = 1
```

Comment:
sensitivity is the feature of a specific mechanism, meantime PLDBudgetAccountant corresponds to the whole pipeline.
More details below on a comment on request_budget

Reply:
I think I have this correctly stored in the mechanism object:

`pld.PrivacyLossDistribution.from_laplace_mechanism(
                    noise / mechanism.weight, sensitivity=mechanism.sensitivity)`